### PR TITLE
Mark quarkus-qute as stable

### PR DESCRIPTION
**Pull request by gastaldi**
Marks `quarkus-qute` as stable

### CODE REVIEW

**Comment by gastaldi** [comment]
Guides need to be changed too

**Comment by mkouba** [comment]
NOTE: this PR should not be merged before https://github.com/quarkusio/quarkus/pull/14990.

**Comment by gastaldi** [comment]
> NOTE: this PR should not be merged before #14990.

LOL, I was just about to comment that in this PR. Thanks

File: docs/src/main/asciidoc/qute-reference.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ include::./attributes.adoc[]
 :sectnums:
 :sectnumlevels: 4
 :toc:
-:extension-status: experimental
+:extension-status: stable
 
 include::./status-include.adoc[]
```
Comment (gsmet) [inline docs/src/main/asciidoc/qute-reference.adoc:16]: I think we just dropped this include for most of the stable extensions. We added it only to non-stable extensions so I would rather just drop it.

Comment (gsmet) [reply docs/src/main/asciidoc/qute-reference.adoc:15]: You need to remove the two `include::./status-include.adoc[]` too :)

Comment (gastaldi) [reply docs/src/main/asciidoc/qute-reference.adoc]: Thanks. Done

**Review by gsmet — APPROVED** [review]
Congrats @mkouba !

**Pull request merged**